.github/workflows/dotnetcoretest.yml

```diff
@@ -28,9 +28,9 @@ jobs:
         run: make
         if: matrix.os == 'ubuntu-latest'
       - name: Setup .NET Core
-        uses: actions/setup-dotnet@v1.8.0
+        uses: actions/setup-dotnet@3
```

Comment:
The ubuntu/windows image have .NET 7.0 pre-installed, should be unnecessary. (but deterministic for sure)

Reply:
I like the explicitness, but good to know.